content/24.vaccines-novel.md

```diff
@@ -159,7 +159,7 @@ Table: DNA vaccines approved in at least one country [@url:https://covid19.track
 #### Plasmid-Vectored DNA Vaccines
 
 Many DNA vaccines use a plasmid vector-based approach, where the sequence encoding the antigen(s) against which an immune response is sought are cultivated in a plasmid and delivered directly to an appropriate tissue [@url:https://www.who.int/biologicals/areas/vaccines/dna/en].
-Plasmids can also be designed to act as adjuvants by encoding molecules that supplement the immune response, such as immune stimulant molecules [@doi:10.1358/dot.2018.54.5.2807864].
+Plasmids can also be designed to act as adjuvants by encoding molecules such as cytokines that supplement the immune response [@doi:10.1358/dot.2018.54.5.2807864].
```

Comment:
@RLordan curious what you think here, the reviewer asked "Which immune stimulant molecules?"
The reference mentions cytokines "and immune stimulant molecules" but does not elaborate.

Reply:
```suggestion
Plasmids can also be designed to act as adjuvants by targeting essential regulators of pathways such as the inflammasome or simply just specific cytokines [@doi:10.1358/dot.2018.54.5.2807864; @doi:[10.1371/journal.pone.0231138](https://doi.org/10.1371%2Fjournal.pone.0231138)].
```
Awesome! I propose changing it to your wording!